harperdb-connect.test.js

```diff
@@ -37,6 +37,8 @@ describe('HarperDBConnect Class', () => {
   })
 
   test('can be instantiated with username, password, and url', async () => {
+    expect.assertions(3)
+    db = await new HarperDBConnect('username', 'password', 'http://mockdb.url/')
```

Comment:
The constructor returns a promise and we should be testing that it resolves properly too.

Try adding something along the lines of:
`await expect(new HarperDBConnect('username', 'password', 'http://mockdb.url/')).resolves.toBeInstanceOf(HarperDBConnect)`
This line may be sufficient to replace line 43. 
Try running `jest --coverage` to make sure it remains at 100%

Reply:
Ahh ok i see the point, but should i remove line 41 ( instantiating HaroerDBConnect) or leave it as it is ? I think it should be removed but want to be sure .

Reply:
Leave it as is. We still want to verify that db.options.url is defined.

Reply:
Fixed the issue, coverage is still 100%.